instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/AwsLambdaTracer.java

```diff
@@ -53,4 +55,8 @@ public Scope startScope(Span span) {
   protected String getInstrumentationName() {
     return "io.opentelemetry.aws-lambda-1.0";
   }
+
+  public void forceFlush() {
+    OpenTelemetrySdk.getTracerProvider().forceFlush().join(1, TimeUnit.SECONDS);
```

Comment:
The OpenTelemetry SDK is in the agent class loader, so won't be accessible directly from auto-instrumentation.

Unless we want to shade and put it in the bootstrap class loader.

I wonder if we will have other use cases like this.

For now(?), maybe add a `forceFlush()` method in `auto-api` (soon to be `javaagent-instrumentation-api`), and register the underlying implementation during agent startup (the implementation needs to live in the agent class loader so it can call the SDK directly).